specs/phase1/shard-transition.md

```diff
@@ -64,7 +62,20 @@ def verify_shard_block_signature(beacon_state: BeaconState,
 
 ```python
 def shard_state_transition(shard_state: ShardState,
-                           block: ShardBlock) -> None:
+                           block: ShardBlock,
+                           validate_message: bool = True,
+                           beacon_parent_state: Optional[BeaconState] = None) -> ShardState:
+    if validate_message:
+        assert beacon_parent_state is not None
+        assert verify_shard_block_message(beacon_parent_state, shard_state, block)
```

Comment:
should we add `verify_shard_block_signature` here? or add it back in `on_shard_block`? 

Doesn't look like that's being used anywhere 😅